src/cmap/conn/wire/test.rs

```diff
@@ -1,50 +0,0 @@
-use bson::Document;
-use tokio::sync::RwLockReadGuard;
-
-use super::message::{Message, MessageFlags, MessageSection};
-use crate::{
-    bson::{doc, Bson},
-    cmap::options::StreamOptions,
-    runtime::AsyncStream,
-    test::{log_uncaptured, CLIENT_OPTIONS, LOCK},
-};
-
-#[cfg_attr(feature = "tokio-runtime", tokio::test)]
-#[cfg_attr(feature = "async-std-runtime", async_std::test)]
-async fn basic() {
```

Comment:
this test seemed to be sufficiently covered by our end-to-end integration tests and is a bit too specific to the implementation, so I just decided to remove it rather than try to accommodate any more changes to it.